### PR TITLE
Home

### PR DESCRIPTION
**Pull request by bossmonkey98**
Homepage Updated
React App - Google Chrome 27-03-2022 22_42_27
React App - Google Chrome 27-03-2022 22_42_34
React App - Google Chrome 27-03-2022 22_42_17

### CODE REVIEW

File: src/components/Navbar/Navbar.css [hunk 1]
```diff
@@ -1,115 +1,94 @@
...
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
-#store-logo{
-    height: 4rem;
-    width: 5rem;
+.store-logo{
+    width: 70px;
```
Comment (rutu14) [inline src/components/Navbar/Navbar.css:20]: Suggestion: Can change the unit to rem.

File: src/pages/Productlisting/ProductListing.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
+.home-page-content{
+    display:flex;
+}
+
+.filters{
+    width: 500px;
```
Comment (rutu14) [inline src/pages/Productlisting/ProductListing.css:6]: Suggestion: Can change to rem

**Review by rutu14 — APPROVED** [review]
The code is good to merge

File: src/components/Navbar/Navbar.css (hunk 1 above)
Comment (sanjitsarkar) [reply src/components/Navbar/Navbar.css:20]: Change it to rem

**Review by sanjitsarkar — APPROVED** [review]
fix the issue mentioned in comments and then your are good to merge.

**Pull request merged**